client/my-sites/stats/activity-log-date/index.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import Button from 'components/button';
+import ActivityLogItem from '../activity-log-item';
+
+class ActivityLogDate extends Component {
```

Comment:
Good point, `ActivityLogDay` makes sense to me.  Will probably want to change the name of the directory as well.